app/assets/javascripts/float-kit/addon/lib/d-menu-instance.js

```diff
@@ -0,0 +1,64 @@
+import { inject as service } from "@ember/service";
+import { action } from "@ember/object";
+import { setOwner } from "@ember/application";
+import { MENU } from "float-kit/lib/constants";
+import { guidFor } from "@ember/object/internals";
+import FloatKitInstance from "float-kit/lib/float-kit-instance";
+
+export default class DMenuInstance extends FloatKitInstance {
+  @service menu;
+
+  constructor(owner, trigger, options = {}) {
+    super(...arguments);
+
+    setOwner(this, owner);
+    options = { ...MENU.options, ...options };
+    this.id = trigger.id || guidFor(trigger);
+    this.trigger = trigger;
+    this.options = options;
+    this.setupListeners();
+  }
+
+  @action
+  onMousemove(event) {
+    if (this.trigger.contains(event.target) && this.expanded) {
+      return;
+    }
+
+    this.onTrigger(event);
+  }
+
+  @action
+  onClick(event) {
+    if (this.expanded && this.untriggers.includes("click")) {
+      this.onUntrigger(event);
+      return;
+    }
+
+    this.onTrigger(event);
+  }
+
+  @action
+  onMouseleave(event) {
```

Comment:
Same minor comment about casing here... really sticks out having stuff like `onMouseleave`...should be `onMouseLeave` IMO. Not dying on any hills, just saying it looks weird

Reply:
yes Im always mixed on this... The js event is `mouseleave` not `mouseLeave` 🤷‍♂️